submitit/local/local.py

```diff
@@ -246,12 +253,14 @@ def start_controller(
         SUBMITIT_STDERR_TO_STDOUT="1" if stderr_to_stdout else "",
         SUBMITIT_EXECUTOR="local",
         CUDA_VISIBLE_DEVICES=cuda_devices,
+        SUBMITIT_LOCAL_WITH_SHELL="1" if setup else "",
     )
     # The LocalJob will be responsible to polling and ending this process.
     # pylint: disable=consider-using-with
-    process = subprocess.Popen(
-        [sys.executable, "-m", "submitit.local._local", str(folder)], shell=False, env=env
-    )
+    proc_cmd: tp.Any = [sys.executable, "-m", "submitit.local._local", str(folder)]
+    if setup:
+        proc_cmd = " && ".join(list(setup) + [" ".join(proc_cmd)])
```

Comment:
this should be a shlex.join to properly quote strings (imagine there is a space in "folder")